### PR TITLE
Upgrade @actions/core to 1.10.0

### PR DESCRIPTION
**Pull request by ymmt2005**
octoken-action used the deprecated `::set-output` workflow command.
The feature will soon be removed, so we have to upgrade @actions/core
to 1.10.0 and recompile the source code.

### CODE REVIEW

**Comment by ymmt2005** [comment]
The test fails because it depends on the implementation of `core.setOutput()`.
The implementation has changed and no longer outputs the deprecated `::set-output`.
https://github.com/cybozu/octoken-action/blob/main/__tests__/Octoken.test.ts#L63-L65

Actually, the test should never reference how the dependency library is implemented.
Instead, it should only check if the code calls the dependency library correctly.

I'm not at all familiar with JavaScript/TypeScript tooling, so I cannot fix it right.
I could only remove the test code, so want to leave it to a maintainer.

**Review by ganta — APPROVED** [review]
I fixed tests in #6 and rebased this branch.
Thank you for your contribution!

**Pull request merged**
